base/methodshow.jl

```diff
@@ -206,24 +202,41 @@ function show(io::IO, m::Method)
         return
     end
     print(io, decls[1][2], "(")
-    join(
-        io,
-        String[isempty(d[2]) ? d[1] : string(d[1], "::", d[2]) for d in decls[2:end]],
-        ", ",
-        ", ",
-    )
+
+    # arguments
+    for (i,d) in enumerate(decls[2:end])
+        printstyled(io, d[1], color=:light_black)
+        print(io, "::")
+        if isempty(d[2])
+            printstyled(io, "Any", color=:bold)
+        else
+            print_type_bicolor(io, d[2], color=:bold, inner_color=:normal)
+        end
+        i < length(decls)-1 && print(io, ", ")
+    end
     kwargs = kwarg_decl(m)
     if !isempty(kwargs)
         print(io, "; ")
-        join(io, map(sym_to_string, kwargs), ", ", ", ")
+        for kw in kwargs
+            skw = sym_to_string(kw)
+            if QuoteNode(kw) in m.roots # then it's required
+                printstyled(io, skw, color=:bold)
+            else
+                print(io, skw)
```

Comment:
This is meant to check which keyword arguments are required, and print those in bold. 

Will this be robust? I don't know any details of the method struct, but in cases I could think of this appears to work.

Reply:
Picture of this, BTW. Left is master, right this PR. 

<img width="976" alt="Screenshot 2021-05-27 at 00 20 00" src="https://user-images.githubusercontent.com/32575566/119765533-7224b900-be81-11eb-9bab-b10e1ef14250.png">

Reply:
no, this is not robust. but quite amusing that you found this

Reply:
Ok. I just poke around... Is there a robust way to query this?

Reply:
I don't know of one right now. Let's leave that for later work?